src/components/RecentlyOpened.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div class="recently-opened">
+    <span>Recently Opened:</span>
+    <div class="recent-items">
+      <a v-for="path in paths" v-bind:title="path.path" :key="path.path" @click="$emit('selected', path.path)">
+        {{ path.display }}
+      </a>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapState, mapActions } from 'vuex';
+
+export default {
+  methods: {
+    ...mapActions(['populateRecentlyOpened'])
+  },
+
+  created() {
+    this.populateRecentlyOpened();
+  },
+
+  computed: {
+    ...mapState(['recentlyOpenedPaths']),
+
+    paths() {
+      const pathsToDisplay = [...this.recentlyOpenedPaths].splice(0, 2); // show only 2 paths
```

Comment:
@emanlodovice Why not `slice` the `recentlyOpenedPaths` array directly instead?

Reply:
because it changes the value stored on state.

Reply:
updated